src/components/PlatformNavigator/index.tsx

```diff
@@ -0,0 +1,88 @@
+import { useState, useRef, useEffect } from 'react';
+import { Button, Flex, Text, View, useTheme, VisuallyHidden } from '@aws-amplify/ui-react';
+import { IconChevron } from '@/components/Icons';
+import { frameworks } from '@/constants/frameworks';
+import { InfoPopover } from './InfoPopover';
+import Link from 'next/link';
+import classNames from 'classnames';
+import { useClickOutside } from '@/utils/useClickOutside';
+
+export function PlatformNavigator({ currentPlatform }) {
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+
+  const { tokens } = useTheme();
+
+  const triggerRef = useRef<HTMLButtonElement>(null);
+
+  const contentRef = useClickOutside((e) => {
+    if (triggerRef.current && !triggerRef.current.contains(e.target)) {
+      if (isOpen) {
+        setIsOpen(false);
+      }
+    }
+  });
+
+  useEffect(() => {
+    if (isOpen) {
+      contentRef?.current?.focus();
+    }
+  }, [isOpen]);
+
+  const platformItem = frameworks.filter((platform) => {
+    return platform.title === currentPlatform;
+  })[0];
+
+  return (
+    <>
+      <View className={`platform-navigator`}>
+        <Text fontWeight={tokens.fontWeights.bold} padding={tokens.space.small}>Choose your framework/language</Text>
```

Comment:
Theres some extra padding we can get rid of and just keep the bottom padding
```suggestion
        <Text fontWeight={tokens.fontWeights.bold} paddingBottom="small">Choose your framework/language</Text>
```